templates/helpers.js

```diff
@@ -402,14 +402,18 @@ DocMapInfo.prototype.getCurrentTree = function(){
     //
     var getChildren = this.getChildren.bind(this),
         getNestedDocObject = this.getNestedDocObject.bind(this);
-
+    
+    var self = this;
     var cur = this.getCurrent();
-
     var curChildren = this.getNestedChildren(cur);
-
     this.getParents(cur, function(docObject){
         curChildren = getChildren(docObject).map(function(docObject){
             if(docObject.name === cur.name) {
+                if(cur.subchildren){
+                    curChildren.forEach(function(child, i){
+                        if(child.docObject) curChildren[i].children = self.getNestedChildren(child.docObject);
```

Comment:
`curChildren[i]` is `child`, right?

Also, this line should be [split on multiple lines](https://contribute.jquery.org/style-guide/js/#spacing).